ci.go

```diff
@@ -89,10 +101,11 @@ func main() {
 	flag.StringVar(&outFile, "output", ".index.json", "File to output index")
 	flag.BoolVar(&generate, "generate", false, "File to output index")
 	flag.StringVar(&inputFile, "input", ".index.json", "File to read index from")
+	flag.BoolVar(&skipDataFiles, "no-data-files", false, "Don't Download data files")
```

Comment:
Shouldn't we have a target "data_file" like for bouncers or configuration file instead of this flag?